ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageSingle.test.js

```diff
@@ -56,16 +56,16 @@ describe('Workload CVE Image Single page', () => {
         const cardSelector = selectors.summaryCard('CVEs by severity');
         cy.get(
             [
-                `${cardSelector} svg + *:contains("Critical")`,
-                `${cardSelector} svg + *:contains("Important")`,
-                `${cardSelector} svg + *:contains("Moderate")`,
-                `${cardSelector} svg + *:contains("Low")`,
+                `${cardSelector} span.pf-v5-c-icon:contains("Critical") ~ p`,
+                `${cardSelector} span.pf-v5-c-icon:contains("Important") ~ p`,
+                `${cardSelector} span.pf-v5-c-icon:contains("Moderate") ~ p`,
+                `${cardSelector} span.pf-v5-c-icon:contains("Low") ~ p`,
```

Comment:
@dvail Does this change to assert text within icon instead of following `p` element solve potential false positive test failure?
![BySeveritySummaryCard](https://github.com/stackrox/stackrox/assets/11862657/77ac1f54-e640-47db-bacd-36b193bf6133)


https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/SummaryCards/BySeveritySummaryCard.tsx#L78